boot/espressif/hal/include/mcuboot_config/mcuboot_config.h

```diff
@@ -45,23 +45,43 @@
  * the default upgrade mode.
  */
 
-/* Uncomment to enable the overwrite-only code path. */
-/* #define MCUBOOT_OVERWRITE_ONLY */
+/* Define to enable the swap-using-move code path. */
+#if defined(CONFIG_BOOT_SWAP_USING_MOVE)
+#define MCUBOOT_SWAP_USING_MOVE 1
+#endif
 
-#ifdef MCUBOOT_OVERWRITE_ONLY
+/* Define to enable the overwrite-only code path. */
+#if defined(CONFIG_BOOT_UPGRADE_ONLY)
+#define MCUBOOT_OVERWRITE_ONLY
 /* Uncomment to only erase and overwrite those primary slot sectors needed
  * to install the new image, rather than the entire image slot. */
 /* #define MCUBOOT_OVERWRITE_ONLY_FAST */
 #endif
 
-/* Uncomment to enable the direct-xip code path. */
-/* #define MCUBOOT_DIRECT_XIP */
+/* Define to enable the direct-xip code path. */
+#if defined(CONFIG_BOOT_DIRECT_XIP)
+#define MCUBOOT_DIRECT_XIP
+#endif
 
 /* Define to enable the ram-load code path. */
 #if defined(CONFIG_BOOT_RAM_LOAD)
 #define MCUBOOT_RAM_LOAD
 #endif
```

Comment:
These are currently undefined modes for Espressif Port, I would like to ask you to add a comment for both and avoid adding them to the .conf files.